src/components/PatchesPage/ListArea.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { ApolloError } from "@apollo/client";
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { Banners } from "components/Banners";
+import { PageWrapper } from "components/styles";
+import {
+  useBannerDispatchContext,
+  useBannerStateContext,
+} from "context/banners";
+import { UserPatchesQuery } from "gql/generated/types";
+import { PatchCard } from "./PatchCard";
+
+export const ListArea: React.FC<{
+  patches?: UserPatchesQuery["user"]["patches"];
+  error?: ApolloError;
+}> = ({ patches, error }) => {
+  const bannersState = useBannerStateContext();
+  const dispatchBanner = useBannerDispatchContext();
+  if (error) {
+    return (
+      <PageWrapper>
+        <Banners
+          banners={bannersState}
+          removeBanner={dispatchBanner.removeBanner}
+        />
+      </PageWrapper>
+    );
+  }
+  if (!patches) {
+    return <StyledSkeleton active title={false} paragraph={{ rows: 4 }} />;
+  }
+  if (patches?.patches.length !== 0) {
+    return (
+      <>
+        {patches?.patches.map(({ commitQueuePosition, ...p }) => (
+          <PatchCard
+            key={p.id}
+            {...p}
+            isPatchOnCommitQueue={commitQueuePosition !== null}
```

Comment:
This expression `commitQueuePosition !== null` can be found in 3 different areas in our codebase. Instead of moving the expression down the component hierarchy, I think we can benefit from making it easily available as part of the Patch type. I made a [ticket](https://jira.mongodb.org/browse/EVG-13618) to try it out